README.md

```diff
@@ -17,8 +17,6 @@ The app is in alpha stage, so it’s time for you to [get involved! 👩‍💻]
 For a roadmap, see our [development milestones](https://github.com/nextcloud/social/milestones) - feel free to pick up any of the tasks which are not yet in progress!
 
 Some requirements in this alpha stage are that your Nextcloud:
-- must use a real SSL certificate
-- must be accessible from the internet
 - must run on the default port
```

Comment:
@daita @violoncelloCH @juliushaertl so can we get rid of this default port requirement? Or is this the case for the rest of Nextcloud anyway? Or is it a non-issue because no one uses other ports in the first place?

We can always just remove it and then if someone opens an issue look into it?

Reply:
Yes, we can get rid of it